src/main/kotlin/org/cascadebot/cascadebot/data/objects/GuildData.kt

```diff
@@ -44,12 +68,15 @@ class GuildData(@field:Id val guildId: Long) {
 
     //region Transient fields
     @Transient
+    @kotlin.jvm.Transient
```

Comment:
Why are these being added to everything? Does the normal Transient annotation not work?

Reply:
The Transient annotation we're using is for polymorphia, the one I added is basically like adding transient to the variable in java

Reply:
Okay. Why do we need this?

Reply:
So the copy of the guild data doesn't this stuff, so we don't diff it.

Reply:
(Although I just realized the new guild data won't end up getting the buttons and stuff copied over, so I'm going to need to account for that)